.travis.yml

```diff
@@ -1 +1,24 @@
 language: go
+
+env:
+  - WITH_COVERAGE=true
+  - GOFLAGS='-race'
+
+go_import_path: github.com/google/monologue
+
+matrix:
+  fast_finish: true
+
+install:
+  - go get -d -t ./...
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.13.2
```

Comment:
Is there a way to put a comment saying that this line is copied from the documentation for integrating golanci-lint with travis?

Reply:
Done.